src/Features.php

```diff
@@ -0,0 +1,106 @@
+<?php
+/**
+ * Class Features.
+ *
+ * @package P4\MasterTheme
+ */
+
+namespace P4\MasterTheme;
+
+/**
+ * Wrapper class for accessing feature settings and setting up the settings page.
+ */
+class Features {
+	/**
+	 * @var string The settings page slug.
+	 */
+	private const PAGE_SLUG = 'planet4_features';
+
+	/**
+	 * @var string The settings title.
+	 */
+	private const PAGE_TITLE = 'Features';
+
+	/**
+	 * @var string The metabox ID.
+	 */
+	private const METABOX_ID = 'planet4_features';
+
+	/**
+	 * @var string The metabox ID.
+	 */
+	public const IMAGE_ARCHIVE = 'feature_image_archive';
+
+	/**
+	 * Features constructor.
+	 */
+	public function __construct() {
+		add_action( 'admin_menu', [ $this, 'add_settings_page' ] );
+	}
+
+	/**
+	 * Check whether a feature is active.
+	 *
+	 * @param string $name The name of the feature we're checking.
+	 *
+	 * @return bool Whether the feature is active.
+	 */
+	public static function active( string $name ): bool {
```

Comment:
`is_active()` might be more clear ? (might also be my french, active means activate 😬 )